### ISSUE TITLE
Unable to write language specific characters in 1.0.3

### ISSUE THREAD

**Issue by mcalster** [getgauge/taiko#1061]
### Expected behavior
Open Google.com and attempt to write æøå in search box.

Expected behvior: The æåø string is written in the search box.

### Actual behavior
Tell us what happens instead
Exception:
Error: Unknown key: "æ"
    at assert (npm\node_modules\taiko\lib\helper.js:83:11)
    at _keyDescriptionForString (npm\node_modules\taiko\lib\handlers\inputHandler.js:81:3)
    at Object._down [as down] (npm\node_modules\taiko\lib\handlers\inputHandler.js:33:23)
    at _write (npm\node_modules\taiko\lib\taiko.js:1455:24)
    at async npm\node_modules\taiko\lib\taiko.js:1430:24
    at async doActionAwaitingNavigation (npm\node_modules\taiko\lib\taiko.js:3410:5)
    at async module.exports.write (npm\node_modules\taiko\lib\taiko.js:1398:3)
    at async module.exports.<computed> (npm\node_modules\taiko\lib\taiko.js:3654:14)
    at async global.<computed> (npm\node_modules\taiko\bin\runFile.js:44:15)

### Versions
```
Working;
Version: 1.0.2 (Chromium:76.0.3803.0)
npm@6.12.1

Not working:
Version: 1.0.3 (Chromium:81.0.3994.0)
npm@6.12.0
```

**Comment by BugDiver** [comment]
@mcalster I'm able to replicate this issue. We will try to prioritize and fix it.

**Comment by srikanthgoudm** [comment]
is this fixed yet?

**Comment by sriv** [comment]
related #1091 

The cause is that taiko refers to a specific keyboard layout for mapping the keycodes to a character as defined in [USKeyboardLayout.js](`https://github.com/getgauge/taiko/blob/master/lib/data/USKeyboardLayout.js`)

This needs to be enhanced to support other keyboard layouts and allow entering other characters (including unicode).

**Comment by SanjuSona-19** [comment]
is there any alternative way to overcome it?

**Comment by mcalster** [comment]
> is there any alternative way to overcome it?

You could downgrade to the previous version, which doesn't have that behavior.

**Comment by SanjuSona-19** [comment]
can you please elaborate?

**Comment by BugDiver** [comment]
@SanjuSona-19 #1096  Should address this. We will try to push a release with this soon.

**Comment by srikanthgoudm** [comment]
Thanks @BugDiver , do we need to update the version?

**Comment by BugDiver** [comment]
@srikanthgoudm The fix is still in the master branch (not released at this moment). So you can install it from the Github master version.

**Comment by zabil** [comment]
Fixed on Taiko 1.0.4
